front/src/components/App/MaintenanceArea/MaintenancePage.tsx

```diff
@@ -0,0 +1,16 @@
+import Button from "../../UI/Button/Button";
+import RouteLink from "../../UI/RouteLink/RouteLink.styled";
+import { APP_PAGE } from "../App";
+
+const MaintenancePage = () => {
```

Comment:
Trochę nam się rozjechało. W założeniu chciałem żeby było MainPage i jakieś Area (SalesArea, MaintenanceArea itd). Ale widzę że to rzeczywiscie jest mylące. Więc jeszcze bym prosił żebyś wszędzie podmienił albo na Page albo na Area - niech wszędzie będzie identycznie (to które wybierzesz już zależy od Ciebie)